src/renderer/components/+extensions/extensions.tsx

```diff
@@ -32,27 +60,264 @@ export class Extensions extends React.Component {
     return extensionDiscovery.localFolderPath;
   }
 
+  getExtensionPackageTemp(fileName = "") {
+    return path.join(os.tmpdir(), "lens-extensions", fileName);
+  }
+
+  getExtensionDestFolder(name: string) {
+    return path.join(this.extensionsPath, sanitizeExtensionName(name));
```

Comment:
Ok, so you suggesting to use it without sanitizing here?

Reply:
Anyway we have to sanitize it since name might contain `/` and currently loading extensions supported only from single level from `~./k8slens/extensions/<folder>`

Reply:
True, but I am thinking more about security. If someone crafted the name to be `@foo\..\..\/blarr` on windows, this would be "sanitized" to `foo\..\..\-blarr` and if we were ever to say "overWrite: true" (which I think I saw in this PR somewhere) then we would be writing to some unknown place.

Reply:
It's not valid name for NPM-package (`@foo\..\..\/blarr`) so no need to worry about it. 
@jakolehm your word?

Reply:
@Nokel81 has a point here.. I think it's possible for an attacker to handcraft a package with invalid name. Of course if user installs a malicious package then it can do even worse things via runtime ... so maybe not super critical to fix here.

Let's create a separate issue about this.